scripts/release-docker.sh

```diff
@@ -10,7 +10,7 @@ set +x
 CLEAR='\033[0m'
 RED='\033[0;31m'
 # Array of valid service names
-VALID_SERVICES=('mina-archive', 'mina-daemon' 'mina-rosetta' 'mina-test-executive' 'mina-batch-txn' 'mina-zkapp-test-transaction' 'mina-toolchain' 'bot' 'leaderboard' 'delegation-backend' 'delegation-backend-toolchain' 'itn-orchestrator')
+VALID_SERVICES=('mina-archive', 'mina-daemon' 'mina-rosetta' 'mina-toolchain' 'bot' 'leaderboard' 'delegation-backend' 'delegation-backend-toolchain' 'mina-test-executive' 'mina-batch-txn' 'mina-zkapp-test-transaction' 'delegation-verify' )
```

Comment:
itn-ochestrator was removed . Is it on purpose?

Reply:
Neither @Smorci nor @Sventimir  made this change. @georgeee did you make this change? Thanks.